docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -1697,8 +1697,8 @@ upper(str)
 
 **Related functions**:
 
-- [initcap](#initcap)
 - [lower](#lower)
```

Comment:
hm, why we have this change? 🤔 its not critical but last revision was better as it sorted

Reply:
Ah, I overlooked the sorting part. You're right. I'll revert it back. Thanks for catching this! 👍